### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by crazysmile11012**
to be eula compliant because we need to do it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,5 @@ While it is possible to use this template in Eclipse it is not recommended. Duri
 When using this template the majority of your mod is developed in the Common project. The Common project is compiled against the vanilla game and is used to hold code that is shared between the different loader-specific versions of your mod. The Common project has no knowledge or access to ModLoader specific code, apis, or concepts. Code that requires something from a specific loader must be done through the project that is specific to that loader, such as the Forge or Fabric project.
 
 Loader specific projects such as the Forge and Fabric project are used to load the Common project into the game. These projects also define code that is specific to that loader. Loader specific projects can access all of the code in the Common project. It is important to remember that the Common project can not access code from loader specific projects.
+
+##this project is not affiliated with mojang or microsoft studios
```
Comment (Negative-light) [inline README.md:28]: Maybe we format this like:

# Disclaimers

This project is not affiliated wit Mojang Studios or Microsoft

**Review by Negative-light — CHANGES_REQUESTED** [review]
Let's swap up the formatting a bit to make it more clear and in case we need to add more disclaimers we can.

**Comment by Negative-light** [comment]
Thanks for noticing. Have a few issues with formatting but nothing much.

**Comment by crazysmile11012** [comment]
il do that

**Comment by crazysmile11012** [comment]
updated readme.md with the requested changes by reviewer

**Review by Negative-light — APPROVED** [review]
Looks good.

Notes for future updates.

We will need to see how this template lands on other projects. Additionally, we should review legal jargon and such for eventual work.

**Comment by Negative-light** [comment]
This Issue seems to be completed. Closing.

**Pull request merged**